ark/utils/example_dataset_test.py

```diff
@@ -1,12 +1,21 @@
 import pathlib
-from typing import Callable, Iterator, Generator
+from typing import Callable, Generator, Iterator
+
 import pytest
-from ark.utils.example_dataset import ExampleDataset, get_example_dataset
+
 from ark.utils import test_utils
+from ark.utils.example_dataset import ExampleDataset, get_example_dataset
 
 
-@pytest.fixture(scope="session", params=["segment_image_data", "cluster_pixels",
-                                         "cluster_cells", "post_clustering"])
+@pytest.fixture(scope="session", params=["segment_image_data",
+                                         "cluster_pixels",
+                                         "cluster_cells",
+                                         "post_clustering",
+                                         "fiber_segmentation",
+                                         "LDA_preprocessing",
```

Comment:
Do each of these need a different example dataset? I think the LDA training and inference notebook is the only one that has specific preprocessing

Reply:
The datasets themselves are the same, they just have different configurations. Each one only uses the minimal amount of data required.

Reply:
Got it, sounds good